### ISSUE TITLE
Were Ethernet layer, IP addresses, and ports masked during data preprocessing?

### ISSUE THREAD

**Issue by Mr-Righter** [ZGC-LLM-Safety/TrafficLLM#35]
Thank you for open-sourcing this work. I have some questions regarding the data preprocessing process for the traffic detection tasks. In the paper, the “Hyper-parameters” section mentions:

> “In traffic detection tasks, we followed [16] to mask the Ethernet layer, IP addresses, and ports to avoid the bias derived by the sensitive meta-information.”

However, when I checked the dataset from Google Drive, I noticed that these fields were not masked. Also, in the code `preprocess/packet_data_preprocess.py`, these fields appear to be retained during preprocessing.

So, I would like to confirm: were these fields included in the training set?
I ask this because I observed a strong correlation between the class labels and certain IP addresses or port numbers, which might lead the model to overfit on this sensitive information.
